onnxruntime/python/tools/tensorrt/perf/build/Dockerfile.tensorrt-perf

```diff
@@ -1,4 +1,5 @@
 FROM onnxruntime-trt
-RUN apt-get update &&\
-    apt-get -y install libprotobuf-dev protobuf-compiler pciutils &&\
-    pip install coloredlogs numpy flake8 onnx Cython onnxmltools sympy packaging
+RUN rm /opt/miniconda/lib/libstdc++.so.6
```

Comment:
curious about this line. can you provide some context behind it?

Reply:
When updating to NVIDIA Tensorrt container from 20.09 to 20.12 I kept getting this error: 
`apt-get: /opt/miniconda/lib/libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /usr/lib/x86_64-linux-gnu/libapt-pkg.so.6.0)` 

The reason is because we have two libstdc++.so.6 files. Onnxruntime dockerfile installs common dependencies with a script producting /opt/miniconda/lib. Nvidia tensorrt container installs /usr/lib/x86_64-linux-gnu/. Nvidia tensorrt container 20.09 only searches for GLIBCXX_3.4.25 while 20.12 container requires GLIBCXX_3.4.26.

This mismatch causes apt-get to find a problem with /opt/miniconda/lib does not contain GLIBCXX_3.4.26. See below.

`strings ./opt/miniconda/lib/libstdc++.so.6 | grep GLIBCXX` shows me 3.4.0-3.4.25
`strings ./usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX ` shows me 3.4.0-3.4.28

Because of the way the docker container is built and run separately, I first tried to alter the path to find /usr/lib/x86_64-linux-gnu/ before /opt/miniconda/lib, because then I could still install the dependencies but only make temporary changes to the path in the image. This was all done in the Dockerfile.tensorrt-perf above, but the changes weren't effective and I still got the above error. 

**PATH** /usr/local/nvidia/bin:/usr/local/cuda/bin:/code/cmake-3.14.3-Linux-x86_64/bin:/opt/miniconda/bin:/opt/tensorrt/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin

The way I could guarantee resolving which libstdc++.so.6 file was found (with minimal interruption) was by removing the /opt/miniconda/lib/libstdc++.so.6 file. 

**Possible Solutions:**
1. The Dockerfile.tensorrt goes back to 20.09 and I make a new dockerfile for perf script 
2. The install_common_deps.sh fetches https://repo.anaconda.com/miniconda/Miniconda3-4.7.12-Linux-x86_64.sh vs Miniconda3-4.5.11-Linux-x86_64.sh but I would need to make sure .vitisai, .migraphx, .openvino, .openvino_csharp all still work with this change.

Reply:
PR looks good overall. Only thing is I wanted to double confirm that the change to install_common_deps.sh is safe since it has potential for broader impact.